packages/@aws-cdk/aws-ecr-assets/lib/image-asset.ts

```diff
@@ -147,6 +147,29 @@ export class DockerImageAsset extends CoreConstruct implements IAsset {
    */
   public readonly assetHash: string;
 
+  /**
+   * The path to the asset, relative to the current Cloud Assembly
+   *
+   * If asset staging is disabled, this will just be the original path.
+   * If asset staging is enabled it will be the staged path.
+   */
+  public readonly assetPath: string;
```

Comment:
Do these all really need to be `public` ? I would be fine if they were `private`, and your feature would still work.

Reply:
updated them